respx/mocks.py

```diff
@@ -297,14 +297,19 @@ def to_httpx_request(cls, **kwargs):
         Create a `HTTPX` request from transport request arg.
         """
         request = kwargs["request"]
+        method = (
+            request.method.decode("ascii").upper()
+            if isinstance(request.method, bytes)
+            else request.method.upper()
+        )
```

Comment:
I note that the new httpx code still uppercases the method, so that's redundant here. I'd just decode from bytes:

```suggestion
        method = (
            request.method.decode("ascii")
            if isinstance(request.method, bytes)
            else request.method
        )
```